DocumentFormat.OpenXml.Tests/FlatOpcTests.cs

```diff
@@ -145,6 +147,28 @@ public void CanCreateFlatOpcPresentationDocuments8()
             }
         }
 
+        [Fact(Skip = "#380")]
+        public void CanCreateFlatOpcSpreadsheetDocumentsDynamically()
+        {
+            using (MemoryStream stream = new MemoryStream())
+            using (SpreadsheetDocument spreadsheet = SpreadsheetDocument.Create(stream, SpreadsheetDocumentType.Workbook))
+            {
+                try
+                {
+                    WorkbookPart workbookPart = spreadsheet.AddWorkbookPart();
+                    workbookPart.Workbook = new Workbook();
+
+                    spreadsheet.Save();
+
+                    string opc = spreadsheet.ToFlatOpcString();
+                }
+                catch (System.Exception e)
+                {
+                    Assert.True(false, e.Message);
```

Comment:
What's this supposed to test? Is it expected to throw an exception? If so, you want to use the pattern:

```csharp
Assert.Throws<Exception>(()=> [expression]);
```

However, I don't think that's the intended behavior - that's just what happens right now. Can you find the expected value here and assert that the contents are that? I'm assuming you can use the work around described in the issue to save the document, reopen the document, and call `ToFlatOpcString()`